### PR TITLE
chore: updates to docs to incorporate product's recommendations

### PR DESCRIPTION
**Pull request by willyogo**
These are the changes we discussed at the retreat.

Main intentions for these changes:
1. Optimize the process for community contributors supporting the development of new features, whether by building themselves, offering bounties for development, or requesting bounties from ShapeShift DAO
2. Offer support and guidance from product workstream in creating feature spec docs and ui/ux that developers to help save time in developing features and give community developers that contribute to large bounties confidence that their finished feature will get merged and deployed to production

### CODE REVIEW

File: bounties.md [hunk 1]
```diff
@@ -26,5 +26,13 @@ We are at the start of building out more resources for contributors. If you are
 Additionally, feel free to attend Engineering Stand-up Every Weekday 17:00 UTC in [Discord](https://discord.gg/shapeshift) #Standup-AllAccess. You can introduce yourself, and we can often find someone to pair up with you if you need assistance for a few minutes after the meeting.
 
 #### Looking to integrate a large feature?
```
Comment (0xApotheosis) [inline bounties.md:28]: We don't lint for this yet, but I'm trying to instigate best practice markdown use across all of our markdown files - add a new line after any heading.

File: bounties.md [hunk 2]
```diff
@@ -26,5 +26,13 @@ We are at the start of building out more resources for contributors. If you are
...
-Please check out this [document](integrators.md)
- 
\ No newline at end of file
+Please check out [Add Features](integrators.md) for guidance on adding a large feature yourself.
+ 
+#### Bounty Instructions for Big Features
```
Comment (0xApotheosis) [inline bounties.md:31]: New line.

File: offer.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Looking to integrate a big feature?
+
+Check out [Add Features](integrators.md) for guidance on adding a large feature yourself.
+
+#### Bounty Instructions for Big Features:
+
+Projects are welcome and encouraged to offer bounties to get their feature integrated by the large and growing community of ShapeShift developers. The product and engineering workstreams have processes and resources defined to provide guidance, support, and code and UX reviews to any individuals or teams who take on a bounty.
```
Comment (0xApotheosis) [inline offer.md:7]: I like to use sentence case when referring to workstreams, i.e. `Product and Engineering workstreams`, as they are nouns.

File: integrators.md [hunk 4]
```diff
@@ -17,22 +19,25 @@ This process is intended to be followed for large technical initiatives and not
...
+4. Obtain approval from Product Workstream on Feature Spec and UI/UX mockups. Depending on bandwidth, the Product Workstream may be able to provide both of these upon request. Alternatively, you can submit draft specs and mockups to the workstream for feedback and approval and can request assistance at any time in the #product-design-public channel in the [ShapeShift Discord](https://discord.gg/shapeshift). To access the channel, select the Product role in the #role-selector channel.
+    1. [Feature Spec Template & Guide](https://www.notion.so/shapeshift/Product-Feature-Spec-Template-17acd69df70e446fa5656fdaaeccf966) & [Examples](https://www.notion.so/shapeshift/Product-and-Creative-Services-Workstream-16fcd5de93334a389adc656a8bedb570)
```
Comment (0xApotheosis) [inline integrators.md:23]: These links are redirects. Use `https://shapeshift.notion.site/shapeshift/Product-and-Creative-Services-Workstream-16fcd5de93334a389adc656a8bedb570` and `https://shapeshift.notion.site/Product-Feature-Spec-Template-17acd69df70e446fa5656fdaaeccf966`.

File: integrators.md [hunk 5]
```diff
@@ -17,22 +19,25 @@ This process is intended to be followed for large technical initiatives and not
...
+    2. [UI/UX Components & Mockup Templates](https://www.figma.com/file/OHbVrAVaV2xQWUsq0nKzGm/ShapeShift-Library?node-id=25%3A576) & [Example](https://invis.io/WF1262ZUU2Q6)
+6. Obtain an approved design document. Integrating teams should create a design document for their planned feature. Previous examples of these documents can be found [here](designs.md). Your document should start out as a Google document that can be used for questions and comments. Once you believe the document is completed, please reach out to @joshf on Discord to schedule a design review meeting. The first half of the meeting will be spent with core engineers individually reading your design document (in silence). The second half of the meeting will be a discussion of questions, comments or any points that need additional clarification or changes. A determination will be made from the core workstream of either approval to move forward or clear next steps to obtain that approval. Once your document is approved, please open a PR to add it to our [docs repo](https://github.com/shapeshift/docs).
```
Comment (0xApotheosis) [inline integrators.md:25]: Nitpick: `Google Document`, as we are referring to the Google product, Document.

Comment (0xApotheosis) [reply integrators.md:25]: Also, is this still @joshf, or is it now @0xdef1cafe?

If it was still Josh, it's now @joshaf 😉

File: SUMMARY.md [hunk 6]
```diff
@@ -3,8 +3,9 @@
 * [ShapeShift DAO](README.md)
 
 ## How to Contribute
```
Comment (0xApotheosis) [inline SUMMARY.md:5]: New line

**Review by 0xApotheosis — CHANGES_REQUESTED** [review]
A few nitpicks, but nice one.

Also, I know you didn't make this typo, but since you're touching the file could you update `GithHub` to `GitHub` in `bounties.md`?

File: integrators.md (hunk 4 above)
Comment (willyogo) [reply integrators.md]: ty ser

File: integrators.md (hunk 5 above)
Comment (willyogo) [reply integrators.md]: fixed both, ty!

**Comment by willyogo** [comment]
thanks for the review @0xApotheosis! I believe all of your suggestions have been implemented and this is good to merge @0xdef1cafe 🙏

**Review by 0xApotheosis — APPROVED** [review]

**Comment by 0xApotheosis** [comment]
Looks good to me, @willyogo!

**Pull request merged**